src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.14.0
+
+[DATE]
+
+This version is compatible with Kubernetes versions 1.24 to 1.27.
+
+### Deprecatation Notice
+
+- As of this versions deploying helm charts through the Okteto UI has been deprecated.
+- Private Registry credentials defined in helm has been deprecated. Private Registry Credentials are now supported natively by Okteto and credentials defined in helm will be automatically migrated. More info [here](ADD_LINK)
+
+### Bug Fixes
+
+- Reduce file permission for okteto-ca-pemstore certificate file in jobs  <!-- 7303 -->
+- Reduce expiration threshold for trial licenses from 60 to 7 days <!-- 7314 -->
+
+### Improvements
+
+- Upgrade ingress-nginx to 1.8.2 (Helm chart version 4.7.2)  <!-- 7295 -->
+- The Okteto CLI used for Pipeline installation has been upgraded to version [2.21.0](https://github.com/okteto/okteto/releases/tag/2.21.0) <!-- 7310 -->
+
+### New Features
```

Comment:
We should include the support for `1.27` of Kubernetes. Even we mention the supported Kubernetes version in the first part of the release notes, I think we should include it in features when a new version is started to be supported